### PR TITLE
Cyborg Plushies!!

### PR DESCRIPTION
**Pull request by MarkusLarsson421**
## About The Pull Request
Adds six cyborg plushies for AIs, Roboticists and other Sillycon lovers.

They currently all do one out of four random cyborg noises.

## Why It's Good For The Game
More plushie representation for non-moths players.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

https://github.com/user-attachments/assets/2e871972-7ba0-48dd-a013-5aef76f41ebf

image

</details>

## Changelog
:cl:
add: Added Cyborg plushies!!
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
15 cyborgs kind of breaks the sound engine, making it a bit weird. Unsure if I should try to fix this or not? Might be out of the scope for this PR.
image

**Comment by BriggsIDP** [comment]
Very cute! Please replace the plushie on the ai core of rad with the piecekeeper!

**Comment by MarkusLarsson421** [comment]
Just got to make the Mediborg inject the user with a tiny amount of morphine, then this PR will be perfect!

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -909,6 +909,82 @@
 			item_list[initial(I.name)] = choice
 	return item_list
 
+/obj/item/toy/plush/sillycons/
```
Comment (PowerfulBacon) [inline code/game/objects/items/plushes.dm]:
```suggestion
/obj/item/toy/plush/sillycons
```

**Review by Rukofamicom — APPROVED** [review]

**Review by PestoVerde322 — APPROVED** [review]

**Review by PestoVerde322 — CHANGES_REQUESTED** [review]
Missing Inhand Dir icons.

immagine

**Pull request closed without merging**